### PR TITLE
Update Introduction to RipQP: change problem example

### PR DESCRIPTION
**Pull request by geoffroyleconte**
**Updates to Introduction to RipQP**

- [ ] There is an issue for this tutorial: Presolve solves the example, so it is not possible to see the logs.
- [ ] Confirm that you have used a tool such as [Grammarly] to verify that the text is correct.
- Wait for the workflow, look at the [preview], and confirm:
  - [ ] The preview has your tutorial listed.
  - [ ] The preview looks like what you want.

[preview]: https://jso.dev/JSOTutorials.jl/
[Grammarly]: https://www.grammarly.com

### CODE REVIEW

**Comment by dpo** [comment]
@abelsiqueira

File: tutorials/introduction-to-ripqp/index.jmd [hunk 1]
```diff
@@ -95,7 +95,7 @@ At every iteration, RipQP solves two linear systems with the default Predictor-C
 
 To save these systems, you can use:
 
-```julia
+```
```
Comment (tmigot) [inline tutorials/introduction-to-ripqp/index.jmd:98]: What is the motivation for this? It feels weird to have block of code that are not executed.

File: tutorials/introduction-to-ripqp/index.jmd [hunk 2]
```diff
@@ -104,7 +104,7 @@ This will save one matrix and the associated two right hand sides of the PC meth
 iteration four.
 Then, you can read the saved files with:
 
-```julia
+```
```
Comment (tmigot) [inline tutorials/introduction-to-ripqp/index.jmd:107]: Same

**Review by tmigot — COMMENTED** [review]
Thanks @geoffroyleconte for updating this.
The CI fails might be because the PR comes from a fork and not a branch of this repo?

**Comment by geoffroyleconte** [comment]
> Thanks @geoffroyleconte for updating this. The CI fails might be because the PR comes from a fork and not a branch of this repo?

Thanks, this was indeed the issue. 
You can merge the new PR https://github.com/JuliaSmoothOptimizers/JSOTutorials.jl/pull/138 and close this one if you agree with the changes.

**Comment by tmigot** [comment]
Superseded by https://github.com/JuliaSmoothOptimizers/JSOTutorials.jl/pull/138#pullrequestreview-1771745803

**Pull request closed without merging**